experiment/scheduler.py

```diff
@@ -772,11 +800,25 @@ def render_startup_script_template(  # pylint: disable=too-many-arguments
         'no_dictionaries': experiment_config['no_dictionaries'],
         'oss_fuzz_corpus': experiment_config['oss_fuzz_corpus'],
         'num_cpu_cores': experiment_config['runner_num_cpu_cores'],
+        'corpus_variant_id': corpus_variant_id,
         'private': experiment_config['private'],
         'cpuset': cpuset,
         'custom_seed_corpus_dir': experiment_config['custom_seed_corpus_dir'],
     }
 
+    if 'seed_sampling' in experiment_config:
+        kwargs['use_seed_sampling'] = True
+
+        kwargs['randomness_seed'] = experiment_config['seed_sampling'] \
```

Comment:
We generally use paren to break up long expressionsthat don't fit, instead of backslashes.